File: MMF/src/app/place.ts
export class Place {
  id: number;
  name: string;
  cordinatesN: number;
  cordinatesE: number;
  temperature: number;
  country: string;
  soilMoisture: number;

  constructor(
    id: number,
    name: string,
    cordinatesN: number,
    cordinatesE: number,
    temperature: number,
    country: string,
    soilMoisture: number
  ) {
    this.id = id;
    this.name = name;
    this.cordinatesN = cordinatesN;
    this.cordinatesE = cordinatesE;
    this.temperature = temperature;
    this.country = country;
    this.soilMoisture = soilMoisture;
  }
}

File: API/WebWeatherApi/Models/Place.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebWeatherApi.Models
{
    public class Place
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public double CoordinatesN { get; set; }
        public double CoordinatesE { get; set; }
        public double Temperature { get; set; }
        public string Country { get; set; }
        public int SoilMoisture { get; set; }
    }
}